schema.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE animals  (
+  id BIGSERIAL PRIMARY KEY NOT NULL,
+  animal_name TEXT NOT NULL,,
```

Comment:
- [ ] Kindly remove one `,`, you have two there. It throws error because of that. I needed to remove it before I could have my way.

### Error
![comma-error](https://github.com/amanuelbelete95/vet-clinic-database/assets/26597432/5aba7908-36ad-4d5e-94b7-a204a1b16183)